CMakeLists.txt

```diff
@@ -381,6 +405,10 @@ if(US_ENABLE_THREADING_SUPPORT AND US_TSAN_LINK_FLAGS)
   set(US_LINK_FLAGS "${US_LINK_FLAGS} ${US_TSAN_LINK_FLAGS}")
 endif()
 
+if(US_ENABLE_CCOV AND US_CXX_CCOV_FLAGS)
+  set(US_LINK_FLAGS "${US_LINK_FLAGS} ${US_CXX_CCOV_FLAGS}")
```

Comment:
```US_CXX_CCOV_FLAGS``` is used to both set the CXX flags and the linker flags. Should the same variable be used? Do the flags for the compiler and linker ever differ?